vispy/visuals/axis.py

```diff
@@ -100,21 +100,22 @@ def __init__(self, pos=None, domain=(0., 1.), tick_direction=(-1., 0.),
         self.tick_direction = np.array(tick_direction, float)
         self.tick_direction = self.tick_direction
         self.scale_type = scale_type
-        self.axis_color = axis_color
-        self.tick_color = tick_color
 
         self.minor_tick_length = minor_tick_length  # px
         self.major_tick_length = major_tick_length  # px
         self.tick_label_margin = tick_label_margin  # px
         self.axis_label_margin = axis_label_margin  # px
 
-        self.axis_label = axis_label
+        self._axis_label_text = axis_label
 
         self._need_update = True
 
-        self._line = LineVisual(method='gl', width=axis_width)
+        self._line = LineVisual(method='gl', width=axis_width, antialias=True,
+                                color=axis_color)
```

Comment:
Should we take this opportunity to switch to 'agg' lines which will always support `width`? The GL line width is not supported on modern GPUs (it is deprecated from the specification).

Reply:
The issue is that the ``LineVisual`` below uses ``connect='segments'`` which doensn't work with agg. We could still change the main axis line but not the ticks to agg, though that might look inconsistent.

Reply:
Wow, I didn't even realize that. I really thought I was using Agg for everything in my own work, but I guess now.